ambry-router/src/main/java/com/github/ambry/router/PutBlobMetaInfo.java

```diff
@@ -101,17 +133,17 @@ public List<Pair<String, Long>> getOrderedChunkIdSizeList() {
    * @return the JSON string representation of the object
    */
   public static String serialize(PutBlobMetaInfo metaInfo) {
-    JSONObject rootObject = new JSONObject();
-
+    ObjectNode rootObject = objectMapper.createObjectNode();
```

Comment:
can we just do 
```
return objectMapper.writeValueAsString(metaInfo);
```

Reply:
As we discussed, the pair key and value name needs to use meaningful naming instead of "first" and "second".